docs/cody/clients/install-vscode.mdx

```diff
@@ -200,6 +200,35 @@ When you have both a repository and files @-mentioned, Cody will search the repo
   <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/context-with-chips-0624.mp4" type="video/mp4" />
 </video>
 
+### @-mention context providers with OpenCtx
+
+<Callout type="info">OpenCtx context providers in Cody is in Experimental stage. If you have feedback or questions, please visit our [support forum](https://community.sourcegraph.com/c/openctx/10).</Callout>
+
+[OpenCtx](https://openctx.org/) is an open standard for bringing contextual info about code into your dev tools. Cody Free and Pro users can use OpenCtx providers to fetch and use context from the following sources:
```

Comment:
let's leave a note saying Enterprise users can also use this experimental feature but with **limited** support. We don't want to lead them on and present a roadmap, but we shouldn't block on them from trying it.

@aramaraju what do you think?

Reply:
Agreed!